tests/lib/rules/curly.js

```diff
@@ -174,6 +174,14 @@ ruleTester.run("curly", rule, {
 			options: ["multi"],
 			languageOptions: { ecmaVersion: 6 },
 		},
+		{
+			code: "if (foo) { const bar = 'baz'; }",
+			options: ["multi"],
+			languageOptions: {
+				sourceType: "module",
+				ecmaVersion: 2026,
+			},
+		},
```

Comment:
```suggestion
		{
			code: "if (foo) { using bar = 'baz'; }",
			options: ["multi"],
			languageOptions: {
				sourceType: "module",
				ecmaVersion: 2026,
			},
		},
		{
			code: "if (foo) { await using bar = 'baz'; }",
			options: ["multi"],
			languageOptions: {
				sourceType: "module",
				ecmaVersion: 2026,
			},
		},
```

I'm guessing this should have been tests with `using` and `await using` to verify changes in `astUtils.areBracesNecessary`.